### DISCUSSION TITLE
Geotiff not displaying

### DISCUSSION THREAD

**Discussion by bucricket** [opengeos/leafmap#446] [Q&A]
Hi,

I apologize if this has been discussed before, but I didn't see it.   I cannot repeat https://leafmap.org/notebooks/05_load_raster/.  Openstreetmaps opens and zooms to where the COG is loaded but I cannot see any geotiff overlay.  I have tried installing leafmap from both conda and pip and I get the same result. I am using python 3.11.3 on an AWS instance.  

Thanks for your help in advance!

**Comment by ccalvocm** [comment]
Greetings @bucricket 

In my case installing localtileserver was needed, !pip install localtileserver. Have my Google Colab example:

https://github.com/ccalvocm/GEE/blob/main/examples/notebooks/05_load_raster.ipynb

![leafmap](https://github.com/opengeos/leafmap/assets/52244870/07d46518-6a08-4a79-a404-d0625e608a13)

`Python version 3.10.11`
`Leafmap version 0.20.3`

Also, I logged in with an authenticated Google Earth Engine user, you may need to do that as well.
Kind regards
